hack/prepare-release.sh

```diff
@@ -1,15 +1,38 @@
 #!/bin/bash -e
+script_dir=$(dirname "$(readlink -f "$0")")
+
 version_type=$1
 old_version=$(hack/version.sh)
 new_version=$(hack/bump-version.sh $version_type)
 commits=$(git log --pretty=format:"* %s" $old_version..HEAD)
+release_notes=$(mktemp)
+
+end() {
+    rm $release_notes
+}
+
+trap end EXIT SIGINT SIGTERM SIGSTOP
 
+release-notes \
+    --format go-template:$script_dir/release-notes.tmpl \
+    --release-version $new_version \
+    --required-author kubevirt-bot \
```

Comment:
What does this stand for? Would not it filter only PRs opened by the bot?

Reply:
Ahh right, we need empty string there https://github.com/kubernetes/release/blob/master/cmd/release-notes/main.go#L159

Reply:
Ahh no at the end at master author is kubevirt-bot since it's squashing the PRs.